_authors/meredith-odea.md

```diff
@@ -0,0 +1,10 @@
+---
+name: meredith-odea
+first_name: Meredith
+last_name: O'Dea
+full_name: Meredith O'Dea
+role: 
+image: "/assets/img/team/meredith-odea.jpg"
+active: true
+# published: true
```

Comment:
@JacOng17 should we remove this unused comment now?

Reply:
Sure thing. Will remove it.